tests/DCPS/Serializer/SerializerTest.cpp

```diff
@@ -1,4 +1,6 @@
 #include "common.h"
+#include "dds/DCPS/dcps_export.h"
+#include "dds/DCPS/DCPS_Utils.h"
```

Comment:
```suggestion

#include <dds/DCPS/DCPS_Utils.h>
```

Why did you include the export header?